desktop/app/menu-bar.js

```diff
@@ -3,21 +3,29 @@ import {resolveImage, resolveRootAsURL} from '../resolve-root'
 import hotPath from '../hot-path'
 import menubar from 'menubar'
 
-let color = 'white'
-let platform = ''
-
-if (process.platform === 'darwin') {
-  color = (systemPreferences && systemPreferences.isDarkMode()) ? 'white' : 'black'
-} else if (process.platform === 'win32') {
-  color = 'black'
-  platform = 'windows-'
-}
+let iconType: 'regular' | 'update' | 'badged' = 'regular'
+
+const getIcon = () => { // eslint-disable-line arrow-parens
```

Comment:
I think this might be cleaner if this took a `iconType: IconType` argument and we didn't have this mutable iconType floating around.